modular_skyrat/modules/customization/modules/mob/dead/new_player/body_markings/body_markings_akula.dm

```diff
@@ -0,0 +1,14 @@
+/datum/body_marking/akula
+	icon = 'modular_skyrat/master_files/icons/mob/body_markings/akula_markings.dmi'
```

Comment:
Won't this cause a weird issue where it'd show up as blank in the markings list? I don't honestly remember how that's normally handled.